<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1119</width>
    <height>845</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="rawDataPlotTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Raw data plot</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="centralLayout">
          <item>
           <layout class="QVBoxLayout" name="rawPlotLayout">
            <item>
             <layout class="QHBoxLayout" name="rawPlotTools">
              <item>
               <widget class="QPushButton" name="rawPlotBackBtn">
                <property name="toolTip">
                 <string extracomment="Back"/>
                </property>
                <property name="text">
                 <string>←</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rawPlotForwardBtn">
                <property name="toolTip">
                 <string extracomment="Forward"/>
                </property>
                <property name="toolTipDuration">
                 <number>-1</number>
                </property>
                <property name="text">
                 <string>→</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="rawStepSizeSpinBox">
                <property name="toolTipDuration">
                 <number>-1</number>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999999999</number>
                </property>
                <property name="value">
                 <number>1000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="rawWindowSizeSpinBox">
                <property name="toolTip">
                 <string extracomment="Window size"/>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>999999999</number>
                </property>
                <property name="value">
                 <number>10000</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="rawWindowStartSpinBox">
                <property name="maximum">
                 <number>999999999</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="downsamplingComboBox">
                <item>
                 <property name="text">
                  <string>No downsampling</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Peak</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Subsample</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mean</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="rawPlotUpdateBtn">
                <property name="toolTip">
                 <string/>
                </property>
                <property name="toolTipDuration">
                 <number>-1</number>
                </property>
                <property name="text">
                 <string>Update</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="rawPlotThresholds">
              <item>
               <widget class="QDoubleSpinBox" name="rawPlotThreshold1">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <double>-99.989999999999995</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="rawPlotThreshold2">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <double>-99.989999999999995</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="rawPlotThreshold3">
                <property name="minimumSize">
                 <size>
                  <width>100</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="minimum">
                 <double>-99.989999999999995</double>
                </property>
                <property name="singleStep">
                 <double>0.100000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="Line" name="line">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="axisShiftLabel">
                <property name="text">
                 <string>Shift X axis</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="axisShiftLabelCh1">
                <property name="text">
                 <string>Channel 1</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="axisCh1ShiftSpinBox">
                <property name="decimals">
                 <number>0</number>
                </property>
                <property name="minimum">
                 <double>-100000000.000000000000000</double>
                </property>
                <property name="maximum">
                 <double>100000000.000000000000000</double>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="axisShiftLabelCh2">
                <property name="text">
                 <string>Channel 2</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="axisCh2ShiftSpinBox"/>
              </item>
              <item>
               <widget class="QLabel" name="axisShiftLabelCh3">
                <property name="text">
                 <string>Channel 3</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="axisCh3ShiftSpinBox"/>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="peakFindingTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Peak finding</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>811</width>
          <height>461</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="leftLayout">
           <item row="15" column="1">
            <widget class="QPushButton" name="findPeaksBtn">
             <property name="text">
              <string>Find Peaks</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QCheckBox" name="currentViewCheckBox">
             <property name="text">
              <string>Current view only</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QDoubleSpinBox" name="peakDistanceCBox">
             <property name="minimum">
              <double>1.000000000000000</double>
             </property>
             <property name="maximum">
              <double>9999999999.989999771118164</double>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QCheckBox" name="invertedCh2Box">
             <property name="text">
              <string>Inverted peaks channel 2</string>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QDoubleSpinBox" name="doublePeakDistanceSpinBox">
             <property name="maximum">
              <double>99999999999.990005493164063</double>
             </property>
            </widget>
           </item>
           <item row="11" column="1">
            <widget class="QCheckBox" name="showFilteredPeaksCheckBox">
             <property name="text">
              <string>Show filtered peaks</string>
             </property>
            </widget>
           </item>
           <item row="13" column="0">
            <widget class="QLineEdit" name="filenameLineEdit">
             <property name="placeholderText">
              <string>peaks.pkl</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="peakWidthLabel">
             <property name="text">
              <string>Peak width</string>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QDoubleSpinBox" name="peakWidthCBox"/>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="doublePeakDistanceLabel">
             <property name="text">
              <string>Double peak distance [ms]</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QCheckBox" name="invertedCh3Box">
             <property name="text">
              <string>Inverted peaks channel 3</string>
             </property>
            </widget>
           </item>
           <item row="12" column="1">
            <widget class="QCheckBox" name="showRemovedPeaksCheckBox">
             <property name="text">
              <string>Show unfiltered peaks</string>
             </property>
            </widget>
           </item>
           <item row="14" column="0">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="4" column="1">
            <widget class="QCheckBox" name="invertedCh1Box">
             <property name="text">
              <string>Inverted peaks channel 1</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="peakDistanceLabel">
             <property name="text">
              <string>Peak distance</string>
             </property>
            </widget>
           </item>
           <item row="13" column="1">
            <widget class="QPushButton" name="savePeaksBtn">
             <property name="text">
              <string>Save peaks</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="rightLayout"/>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="waveletFilteringTab">
       <attribute name="title">
        <string>Wavelet filtering</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="waveletLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="1">
             <widget class="QComboBox" name="waveletFamilyCBox">
              <property name="placeholderText">
               <string>Select wavelet family</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="waveletBackBtn">
              <property name="text">
               <string>←</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QComboBox" name="selectChannelCBox">
              <property name="currentText">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Select a channel</string>
              </property>
              <item>
               <property name="text">
                <string>Channel 1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Channel 2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Channel 3</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="QPushButton" name="applyWaveletBtn">
              <property name="text">
               <string>Apply wavelet filter</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="waveletForwardBtn">
              <property name="text">
               <string>→</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="refreshWaveletPlotBtn">
              <property name="text">
               <string>Refresh plot</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="selectWaveletCBox">
              <property name="placeholderText">
               <string>Select wavelet</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1119</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
